index.html

```diff
@@ -15,10 +15,10 @@
 <body>
   <header class="header-container">
     <div class="logo">
-      <p class="main-logo">Julio Quezada.</p>
+      <p class="main-logo">Julio Quezada</p>
     </div>
     <nav class="navigation-bar">
-      <div class="hamburgerMenu">
+      <div class="hamburgerMenu" aria-label="Navigation links">
```

Comment:
- Interaction: The navigation menu and form have appropriate labels and are easily operable with a keyboard. However, the hamburger menu icon could benefit from an aria-label to describe its function for screen reader users. 
-- Changes suggested.
-- We fixed it.